sections/preliminaries.tex

```diff
@@ -0,0 +1,136 @@
+For each topic $k \in \{1,\dots, K\}$ a topic-word distribution $\beta_k$ is sampled from a V-dimensional Dirichlet distribution parameterized by $\eta$.
+That is, K topics $\beta_{1:k}$ are sampled, each being a distribution over the vocabulary, written as: $\beta_k \sim Dirichlet(\eta)$.
+Likewise, for each document $d \in \{1,\dots, D\}$ a document-topic distribution $\theta_d$ is sampled from a K-dimensional Dirichlet distribution parameterized by $\alpha$.
+For each word $n \in \{1, \dots, N_d\}$ in each document $d$, a topic $z_{d,n}$ is sampled from a K-multinomial distribution $\theta_d$, and then a word $w_{d,n}$ is sampled from a V-multinomial distribution $\beta_{z_{d,n}}$.
+The generative process for each document is seen in these steps:
+
+\vspace{\topsep}
+\begin{enumerate}
+	\item Draw topic proportion $\theta_d \sim Dirichlet(\alpha)$
+	\item For each word $n$ in the document:
+	\begin{enumerate}
+		\item Draw topic assignment $z_{d,n} \sim Mult(\theta_d)$
+		\item Draw word $w_{d,n} \sim Mult(\beta_{z_{d,n}})$
+	\end{enumerate}
+\end{enumerate}
+\vspace{\topsep}
+
+The generative process for topics and documents, generates a list of $K$ topics and $D$ documents that can be used as a $K \times V$ matrix of topic-word distributions and a $D \times K$ matrix of document-topic distributions, respectively.
+The plate notation for \gls{lda} can be seen in \autoref{fig:standard_lda}.
+
+\input{figures/lda_plate}
+
+After training the \gls{lda} model, there are multiple possibilities for exploring the corpus using the posterior distributions of the hidden random variables.
+One possibility is to visualize the posterior topics of the model, e.g., by sorting $\beta_k$ according to the highest probabilities. \vejleder{gerne mere præcist}
+It is also possible to visualize the documents by, e.g., sorting by the highest topic proportions in $\theta_d$.
+Another possibility of exploration is finding similar documents by using a distribution distance function on the topic proportions $\theta_d$ between documents~\cite{blei2009topic}.
+
+
+\subsection{Author-Topic \gls{lda}}\label{subsec:auth_prelim} \vejleder{introducer modellen som den er i \cite{author_topic_2012}, gerne flere detaljer}
+\citet{author_topic_2012} present the Author-Topic model.
+It seeks to find topics based on author metadata, and is based on the assumption that authors prefer to write about specific topics.
+In this model, there are no document-topic distributions $\theta$, but rather one author-topic distribution for each author.
+
+This is a \gls{um}, meaning that topics are only word distributions and are chosen based on the data of the documents, rather than \glspl{dm} where topics are fitted to have an influence on both word distributions and other metadata. \todo{har vi lyst til at skulle beskrive upstream og downstream for alle modeller? }
```

Comment:
```suggestion
This is an \gls{um}, meaning that topics are only word distributions and are chosen based on the data of the documents, rather than \glspl{dm} where topics are fitted to have an influence on both word distributions and other metadata. \todo{har vi lyst til at skulle beskrive upstream og downstream for alle modeller? }
```

Reply:
```suggestion
```